src/components/patient/Orders.js

```diff
@@ -1,34 +1,429 @@
+                <ToggleButtonGroup
+                  value={dose}
+                  exclusive
+                  onChange={doseChange}
+                >
+                {tempMed ? tempMed.fields.dose.map((d) => (
+                  d.map((sub) => (
+                    createToggleButton(sub, sub)
+                  ))
+                )) : ''}
+              </ToggleButtonGroup>
+
+              <p>Frequency: </p> 
+                <ToggleButtonGroup
+                  value={freq}
+                  exclusive
+                  onChange={freqChange}
+                >
+                {createToggleButton(tempMed ? tempMed.fields.frequency : '', tempMed ? tempMed.fields.frequency : '')}
+              </ToggleButtonGroup>
+
+              <p>Refills: </p> 
+                <ToggleButtonGroup
+                  value={refill}
+                  exclusive
+                  onChange={refillChange}
+                >
+                {createToggleButton(tempMed ? tempMed.fields.refills : 0, tempMed ? (tempMed.fields.refills) : refill)}
+              </ToggleButtonGroup>
+
+          <div style={{backgroundColor: "blue", height: '40px'}}>
+            <div style={{float: "right"}}>
+              <Button onClick={() => { closeDose(true) }}><CheckIcon color="success"/>Accept</Button>
+              <Button onClick={() => { closeDose(false) }}><ClearIcon color="error"/>Cancel</Button></div>
+          </div>
+
+        </Dialog>
+        <Dialog fullWidth maxWidth="md" onClose={() => { setOpenLab(false) }} open={openLab}>
+
+          <div style={{backgroundColor: "blue", height: "40px"}}>
+            CBC w/ DIFF
+            <DialogActions style={{float: "right"}}>
+              <Button onClick={() => { closeLab(true) }}><CheckIcon color="success"/>Accept</Button>
+              <Button onClick={() => { closeLab(false) }}><ClearIcon color="error"/>Cancel</Button>
+            </DialogActions>
+          </div>
+
+          <div>
+              Order type: 
+                <ToggleButtonGroup
+                  value={type}
+                  exclusive
+                  onChange={typeChange}
+                >
+                  {createToggleButton('outpatient', (<HomeIcon/>))}
+                  {createToggleButton('inpatient', (<HotelIcon/>))}
+                </ToggleButtonGroup>
+
+              <p>Status: </p> 
+              <ToggleButtonGroup
+                value={status}
+                exclusive
+                onChange={statusChange}
+              >
+                {createToggleButton('normal', 'Normal')}
+                {createToggleButton('standing', 'Standing')}
+                {createToggleButton('future', 'Future')}
+              </ToggleButtonGroup>
+
+              {status==='standing' && (<><p>Interval: </p>
+              <ToggleButtonGroup
+                value={interval}
+                exclusive
+                onChange={intervalChange}
+              >
+                {createToggleButton(30, '1 Month')}
```

Comment:
I would just inline the `ToggleButton` component here instead of calling out to a function, it can be a bit confusing this way.

Reply:
removed togglebutton function in commit